src/main/java/com/github/magickoders/jar/EntityBuilder.java

```diff
@@ -102,7 +102,7 @@ public Entity build() {
     Objects.requireNonNull(type, "No value set for type");
     Objects.requireNonNull(stereotype, "No value set for stereotype");
 
-    var entity = new Entity(modifiers, type, stereotype, fields, methods);
+    final var entity = new Entity(modifiers, type, stereotype, fields, methods);
```

Comment:
final is useless here, if you want all local variables to be final, configure your IDE for that but adding final make the code less readable with not a lot of bangs for bucks

Reply:
Is configuring the IDE enough even if there was this error message (see below) in the previous build attempt ?

Error:  /home/runner/work/umldoc/umldoc/src/main/java/com/github/magickoders/jar/EntityBuilder.java:105:5: Distance between variable 'entity' declaration and its first usage is 6, but allowed 3.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value). [VariableDeclarationUsageDistance]

Reply:
stop setting the fields to null will solve the issue